src/qibolab/instruments/erasynth.py

```diff
@@ -107,8 +114,17 @@ def close(self):
 
 
 class ERA(LocalOscillator):
-    def __init__(self, name, address, ethernet=True, reference_clock_source=None):
-        super().__init__(name, address, reference_clock_source)
+    """Driver to control the ERAsynth++ local oscillator.
+
+    This driver is using:
+    https://qcodes.github.io/Qcodes_contrib_drivers/_modules/qcodes_contrib_drivers/drivers/ERAInstruments/erasynth.html#ERASynthBase.clear_read_buffer
```

Comment:
Maybe:
https://qcodes.github.io/Qcodes_contrib_drivers/api/generated/qcodes_contrib_drivers.drivers.ERAInstruments.html#qcodes_contrib_drivers.drivers.ERAInstruments.erasynth.ERASynth

(that seems a random function)